src/pages/PMT_PaymentWizard.page

```diff
@@ -240,112 +299,79 @@
                     </div>
                 </div>
 
                 <!-- START PAYMENT WRITEOFF  -->
                 <apex:outputText rendered="{!AND(hasPaymentAccess, NOT(ISNULL(PaymentsSettings)), PaymentsSettings.npe01__Payments_Enabled__c == true, isWriteoffWizard == true)}">
-                    <div class="slds-section-title_divider" id="pmtWizardTitle2">
-                        <h2><apex:outputText value="{!$Label.pmtWizardSectionTitle}"/></h2>
-                    </div>
-                    <apex:outputText rendered="{!NOT(ISNULL(currentOpp))}">
-                        <div class="slds-grid slds-wrap slds-form_horizontal">
-                            <div class="slds-col slds-col_padded">
-                                <div class="slds-form-element">
-                                    <label class="slds-form-element__label" for="opplink2">{!$ObjectType.Opportunity.label}</label>
-                                    <div class="slds-form-element__control">
-                                        <span class="slds-form-element__static">
-                                            <a href="{!URLFOR($Action.Opportunity.View, currentOpp.id)}">
-                                                {!currentOpp.Name}
-                                            </a>
-                                        </span>
-                                    </div>
-                                </div>
-                                <div class="slds-form-element">
-                                    <label class="slds-form-element__label" for="oppStage2">{!$ObjectType.Opportunity.Fields.StageName.Label}</label>
-                                    <div class="slds-form-element__control">
-                                        <span class="slds-form-element__static">
-                                            <apex:outputField value="{!currentOpp.StageName}" id="oppStage2"/>
-                                        </span>
-                                    </div>
-                                </div>
-                                <div class="slds-form-element">
-                                    <label class="slds-form-element__label" for="oppCloseDate3">{!$ObjectType.Opportunity.Fields.CloseDate.Label}</label>
-                                    <div class="slds-form-element__control">
-                                        <span class="slds-form-element__static">
-                                            <apex:outputField value="{!currentOpp.CloseDate}" id="oppCloseDate3"/>
-                                        </span>
-                                    </div>
-                                </div>
-                            </div>
+
+                    <apex:outputText rendered="{!AND(NOT(ISNULL(CurrentOpp)), haveAmount, NOT(ISNULL(PaymentsSettings)), PaymentsSettings.npe01__Payments_Enabled__c == true)}" 
```

Comment:
Looks like this was like this before, but `CurrentOpp` should be lower case to match the var the controller